Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -1129,23 +1132,23 @@ This section elaborates on the various ``kubeProxyReplacement`` options:
   start the NodePort health check server (``kube-proxy`` will also attempt to start
   this server, and there would otherwise be a clash when cilium attempts to bind its server to the
   same port). A few example configurations
-  for the ``partial`` option are provided below.
+  for the ``false`` option are provided below.
 
 .. note::
 
-    Switching from the ``strict`` to ``disabled`` mode, or vice versa can break
+    Switching from the ``true`` to ``false`` mode, or vice versa can break
     existing connections to services in a cluster. The same goes for enabling, or
     disabling ``socketLB``. It is recommended to drain all the workloads before
     performing such configuration changes.
 
-  The following Helm setup below would be equivalent to ``kubeProxyReplacement=strict``
+  The following Helm setup below would be equivalent to ``kubeProxyReplacement=true``
```

Comment:
What if users disable some components with `kubeProxyReplacement=true`? Is it acceptable? 
I think some users might mistakenly do this because they don't want to disable the kube-proxy replacement. (They want to enable it partially)

For example,
```
helm install cilium |CHART_RELEASE| \\
    --namespace kube-system \\
    --set kubeProxyReplacement=true \\
    --set hostPort.enabled=false
```

Reply:
> What if users disable some components with kubeProxyReplacement=true? Is it acceptable?

Noup, it's not possible. The `kube-proxy Hybrid Modes` section explains that. Do you think that we need to elaborate more in that section?

Reply:
I see.

> Do you think that we need to elaborate more in that section?

I think we don't need it. Indeed, users will be able to understand if they read the `kube-proxy Hybrid Modes` section

Reply:
Ideally, we should rework our KPR docs in v1.15.